mysql-test/suite/rpl/t/optimized_replace_mixed_engine.test

```diff
@@ -0,0 +1,55 @@
+--source include/have_log_bin.inc
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
```

Comment:
Move `--source include/master-slave.inc` to the very end of the `--source` section (after all lightweight `have_xxx.inc`)
Currently, because of the `have_debug.inc` this test will be skipped in Release configuration but this will happen only after a long-lasting master-slave initialization process.